.github/workflows/release.yml

```diff
@@ -6,21 +6,24 @@ on:
       - 'v*.*.*'
   workflow_dispatch:
 
+env:
+  SUBWASM_VERSION: 0.20.0
+
 jobs:
   check-version:
     runs-on: ubuntu-latest
     outputs:
       proceed: ${{ steps.check_version.outputs.proceed }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4.1.1
       - id: check_version
         run: |
           VERSION=$(grep -m1 version Cargo.toml | sed -E 's/.*"(.*)".*/\1/')
           if [ "v$VERSION" == "${{ github.ref_name }}" ]; then
-            echo "::set-output name=proceed::true"
+            echo "proceed=true" >> $GITHUB_OUTPUT
           else
             echo "Version in Cargo.toml ($VERSION) does not match current tag ${{ github.ref_name }}. Canceling release..."
-            echo "::set-output name=proceed::false"
+            echo "proceed=false" >> $GITHUB_OUTPUT
```

Comment:
`check-version` has to fail.
When that's fixed `proceed=true` can be removed

Reply:
Fixed by making the job exit when condition is not met